pkg/accounts/service/authenticate.ts

```diff
@@ -8,7 +8,9 @@ import type { TokenGenerator } from './tokenGenerator.js';
 
 export interface TokenPair {
   authorizationToken: string;
+  authorizationTokenExpiresIn: number;
   refreshToken: string;
+  refreshTokenExpiresIn: number;
```

Comment:
```suggestion
  authorizationTokenExpiresIn: PulsateTime;
  refreshToken: string;
  refreshTokenExpiresIn: PulsateTime;
```
`PulsateTime` 型が適切かと思います.

Reply:
`ExpiresIn`系はJWTをパースすれば(PulsateTimeではなくUNIXMilli)求めることができるので廃止する方針でいます